internal/inspect/inspect.go

```diff
@@ -111,27 +142,38 @@ func startRPCServers(ctx context.Context, cfg *config.RPCConfig, logger log.Logg
 			Addr:    listenerAddr,
 		}
 		if cfg.IsTLSEnabled() {
-			keyFile := cfg.KeyFile()
-			certFile := cfg.CertFile()
-			listenerAddr := listenerAddr
+			var (
+				keyFile      = cfg.KeyFile()
+				certFile     = cfg.CertFile()
+				listenerAddr = listenerAddr
+			)
 			g.Go(func() error {
-				logger.Info("RPC HTTPS server starting", "address", listenerAddr,
-					"certfile", certFile, "keyfile", keyFile)
+				logger.Info(
+					"RPC HTTPS server starting",
+					"address", listenerAddr,
+					"certfile", certFile,
+					"keyfile", keyFile,
+				)
+
 				err := server.ListenAndServeTLS(tctx, certFile, keyFile)
 				if !errors.Is(err, net.ErrClosed) {
 					return err
 				}
+
```

Comment:
```suggestion
```

Reply:
Btw the new lines are to avoid having too many line changes but if it is refactoring done automatically then ignore them.

Reply:
I add the empty lines manually because I think it makes the code a bit more readable. Sometimes I feel it is too packed for comfortable parsing, but maybe it's just me 😅